sway-ir/tests/ir_to_ir/constants_contract_calls.in_ir

```diff
@@ -0,0 +1,61 @@
+script {
+    fn main() -> u64 {
+        local ptr u64 a
+        local mut ptr { b256 } args_struct_for_get_b256
+        local mut ptr { u64, b256 } args_struct_for_get_s
+        local mut ptr { u64 } args_struct_for_get_u64
+        local ptr b256 b
+        local ptr { u64, b256 } s
+
+        entry:
+        v0 = get_ptr mut ptr { u64 } args_struct_for_get_u64, ptr u64, 0
+        v1 = const u64 1111
+        v2 = insert_value v0, { u64 }, v1, 0
+        v3 = const { b256, u64, u64 } { b256 undef, u64 undef, u64 undef }
+        v4 = const b256 0x0c1c50c2bf5ba4bb351b4249a2f5e7d86556fcb4a6ae90465ff6c86126eeb3c0
+        v5 = insert_value v3, { b256, u64, u64 }, v4, 0
+        v6 = const u64 2559618804
+        v7 = insert_value v5, { b256, u64, u64 }, v6, 1
+        v8 = insert_value v7, { b256, u64, u64 }, v2, 2
+        v9 = const u64 0
+        v10 = const b256 0x0000000000000000000000000000000000000000000000000000000000000000
+        v11 = const u64 10000
+        v12 = contract_call v8, v9, v10, v11
+        v13 = get_ptr ptr u64 a, ptr u64, 0
+        store v12, ptr v13
+        v14 = get_ptr mut ptr { b256 } args_struct_for_get_b256, ptr u64, 0
+        v15 = const b256 0x3333333333333333333333333333333333333333333333333333333333333333
+        v16 = insert_value v14, { b256 }, v15, 0
+        v17 = const { b256, u64, u64 } { b256 undef, u64 undef, u64 undef }
+        v18 = const b256 0x0c1c50c2bf5ba4bb351b4249a2f5e7d86556fcb4a6ae90465ff6c86126eeb3c0
+        v19 = insert_value v17, { b256, u64, u64 }, v18, 0
+        v20 = const u64 1108491158
+        v21 = insert_value v19, { b256, u64, u64 }, v20, 1
+        v22 = insert_value v21, { b256, u64, u64 }, v16, 2
+        v23 = const u64 0
+        v24 = const b256 0x0000000000000000000000000000000000000000000000000000000000000000
+        v25 = const u64 20000
+        v26 = contract_call v22, v23, v24, v25
```

Comment:
If we want to add the method name for readability then it could just be an ignored identifier parameter, like `_ = contract_call method_name, v22, v23, v24, v25`.

Or if we want to add it to be useful for tooling then it could be added as metadata.